### DISCUSSION TITLE
ClamAV in external server

### DISCUSSION THREAD

**Discussion by mixman68** [docker-mailserver/docker-mailserver#2539] [Q&A]
Hello, we have a huge ClamAV pool into your infrastructure, and today we use ispconfig and custom tweaks to manage mailbox with LDAP.

I want to migrate domains to new infrastructure with mailserver docker

how I can configure it to use existing ClamAV ?

The physical machine for mailserver doesn't have enough ram to support clamav built-in.

Thanks

**Comment by polarathene** [comment]
I don't know enough about ClamAV integration to know how that'd work with an external one. I assumed it was meant to be coupled with the mailserver to check mail as it arrives, but perhaps there's a way to configure it to use an external server?

What you can do if you've got around 1GB RAM is to configure zram on the system. That can compress RAM which is quite helpful and I was able to use that to build the Dockerfile and run tests on a 1GB VPS in the past. There's also leveraging disk space with swap, or a mix between the two with zswap. ZSwap will provide a compressed cache in RAM of a size such as 20% RAM, with any excess being paged out to disk based swap, while the RAM cache keeps any frequently accessed swap pages to avoid the slower disk I/O. It should work too if zram isn't sufficient (you only use one btw).
